TramsDataApi.Test/Controllers/AcademyTransferProjectsControllerTests.cs

```diff
@@ -202,8 +203,10 @@ public void IndexAcademyTransferProject_ReturnsEmptyList_WhenThereAreNoAcademyTr
             var indexAcademyTransferProject = new Mock<IIndexAcademyTransferProjects>();
 
             indexAcademyTransferProject
-                .Setup(get => get.Execute(1))
-                .Returns(() => new List<AcademyTransferProjectSummaryResponse>());
+               .Setup(get => get.Execute(1))
+               .Returns(() =>
+                  Tuple.Create<IList<AcademyTransferProjectSummaryResponse>, int>(
```

Comment:
Small thing -
you can shorten this to - (new List<AcademyTransferProjectSummaryResponse>(), 0))

Reply:
Gave that (and a few variants) a go. That then doesn't conform to the return type, so won't compile :(

Reply:
Fair enough.